xdsl/dialects/onnx.py

```diff
@@ -26,6 +26,54 @@
 from xdsl.utils.exceptions import VerifyException
 
 
+def verify_shapes(
+    lhs_type: TensorType[Attribute] | list[int],
+    rhs_type: TensorType[Attribute] | list[int],
+    res_type: TensorType[Attribute],
+) -> None:
+    """
+    https://github.com/onnx/onnx/blob/main/docs/Broadcasting.md
```

Comment:
Thanks for linking the doc, but could you also add a little blob describing what this method does and leave the URL at the bottom of the docstring for anyone who wants to have a further look?